index.html

```diff
@@ -9,184 +9,193 @@
-                <li><a href="https://github.com/NabilHY"><img class="icons" src="github.png" alt="socialmediaicon"></a>
+                <li><a href="https://medium.com/"><img class="icons" src="images/medium.png" alt="socialmediaicon"></a></li>
+                <li><a href="https://github.com/NabilHY"><img class="icons" src="images/github.png" alt="socialmediaicon"></a>
                 </li>
-                <li><a href="https://angel.co/"><img class="icons" src="angellist.png" alt="socialmediaicon"></a></li>
+                <li><a href="https://angel.co/"><img class="icons" src="images/angellist.png" alt="socialmediaicon"></a></li>
             </ul>
-        </section>
-        <div>
-            <button class="button1">Get My resume</button>
+            <button type="button">Get My resume</button>
+
+            <div class="languages">Languages <i class="fa-solid fa-angle-down"></i>
+              <div  class="languages"><img src="images/javascript.png" alt="Javascript">
+              <p>Javascript</p></div>
+              <div  class="languages"><img src="images/html.png" alt="HTML">
+               <p>HTML5</p></div>
+              <div  class="languages"><img src="images/css.png" alt="CSS">
+                 <p>CSS3</p></div>
+              </div>
+
+
+
+        
+        <div class="languages">Frameworks<i class="fa-solid fa-angle-down"></i>
+            
         </div>
-        <section>
-            <ul class="lists">
-                <li class="layout collapse">
-                    <div class="textlayout">Languages</div><img class="vector" src="vector1.png" alt="vector">
-                </li>
-                <li class="layout frame"><img class="pad" src="javascript.png" alt="js">
-                    <div class="textLi">Javascript</div>
-                </li>
-                <li class="layout frame"><img class="pad" src="html.png" alt="html">
-                    <div class="textLi">HTML</div>
-                </li>
-                <li class="layout frame"><img class="pad" src="css.png" alt="css">
-                    <div class="textLi">CSS</div>
-                </li>
-            </ul>
-        </section>
-        <section>
-            <ul>
-                <li class="layout collapse">
-                    <div class="textlayout">Frameworks</div><img class="vector1" src="vector1.png" alt="vector">
-                </li>
-            </ul>
-        </section>
-        <section>
-            <ul>
-                <li class="layout collapse">
-                    <div class="textlayout">Skills</div><img class="vector2" src="vector1.png" alt="vector">
-                </li>
-            </ul>
-        </section>
-        <section>
 
-        </section>
+
+        <div class="skills">Skills<i class="fa-solid fa-angle-down"></i>
+            
+        </div>
+
+    </section>
+
 
 
+    <section class="contact-me">
+        <h2>Contact me</h2>
```

Comment:
- [ ] This text should be centered.

Please, see my comment and the screenshots above about the Figma template.